catalogs/sources/gtfs/realtime/us-california-santa-cruz-metro-scmtd-gtfs-rt-v-p-1910.json

```
@@ -0,0 +1,13 @@
+{
+    "mdb_source_id": 1910,
+    "data_type": "gtfs-rt",
+    "entity_type": "vp",
```

Convert the entity_type to an array:
```suggestion
    "entity_type": ["vp"],
```